index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portifolio skeleton</title>
+  <link rel="stylesheet" href="index.css" />
+</head>
+
+<body>
+  <div class="font">
+    <section class="first-section">
+      <div class="first-container">
+        <div class="font">
+          <header class="header">
+            <a class="my-logo">My Logo</a>
+            <nav class="hamburger">
+              <ul class="ul-style">
+                <li>
+                  <a href="#">
+                    <img src="./images/Union.svg" alt="union background" />
+                  </a>
+                </li>
+              </ul>
+            </nav>
+          </header>
+
+          <section class="section1">
+            <h2 class="first-para">
+              I’m Mirela<br />
+              Glad to see <br />
+              you!
+            </h2>
+            <h2 class="second-para">
+              I’m a software developer! I can help you build a product , feature or
+              website Look through some of my work and experience! If you like what
+              you see and have a project you need coded, don’t hestiate to contact
+              me.
+            </h2>
+            <p class="third-para">LET’S CONNECT</p>
+
+          </section>
+          <div class="social_links">
+            <img src="./images/Vector-2.jpg" class="logo-icons" alt="union" />
+            <img src="./images/Linkedin-icon.jpg" class="logo-icons" alt="union" />
+            <img src="./images/Vector-3.jpg" class="logo-icons" alt="union" />
+            <img src="./images/Vector-4.jpg" class="logo-icons" alt="union" />
+            <img src="./images/Vector.jpg" class="logo-icons" alt="union" />
+          </div>
```

Comment:
- [ ] As requested by the previous reviewer, it would be nice if you place the social media icons in a list item element. Conventionally, items that display in a list fashion should make use of the `ul` and `li` elements
- Hints:
- You can change the `div` that has the class social_links to a `ul` tag
- And then place each social icon image in a `li` 
```mdx
<ul>
  <li> <img src="./images/Vector-2.jpg" class="logo-icons" alt="union" /> </li>
</ul>
```

That will solve it. 👍.  You can adjust your styles after, but it's required you place them in ul - li tags.